src/rkc_utils.jl

```diff
@@ -126,27 +153,26 @@ function maxeig!(integrator, cache::OrdinaryDiffEqMutableCache)
   end
   return false
 end
-
 """
-    choosedeg!(cache) -> nothing
+choosedeg!(cache) -> nothing
```

Comment:
Why the indentation change?

Reply:
I auto-indented it. I'll change it back.

Reply:
PRs for fixing indentation are good, but muddling the diff so it's hard to see what you did in a PR that's about adding/changing an algorithm is bad.

Reply:
Yeah, I'll make sure it never happens again.